src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -568,7 +569,7 @@ case class JesBackend(actorSystem: ActorSystem)
       status match {
         case Run.Success(events) if backendCall.runtimeAttributes.failOnStderr && stderrLength.intValue > 0 =>
           // returnCode will be None if it couldn't be downloaded/parsed, which will yield a null in the DB
-          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption).future
+          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption map ScriptReturnCode).future
```

Comment:
I thought you might say that :p For a "utility" conversion like that it's typically the kind of conversion I don't really need/care to know that it's happening but I hear your point.
I'll make a function and see if I can make use of it in the other cases.